src/gc/pal/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
Yes! (BTW: The CoreRT platform-specific stuff is done some way https://github.com/dotnet/corert/tree/master/src/Native/Runtime)